bedrock/mozorg/templates/mozorg/contribute/tasks/follow-mozilla.html

```diff
@@ -0,0 +1,41 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/tasks/taskview-base.html" %}
+
+{% add_lang_files "mozorg/contribute/signup" %}
+
+{% block page_title %}{{ _('Tweet to @StartMozilla and see how your technical skills can contribute') }}{% endblock %}
+
+{% block more_tasks %}{{ url('mozorg.contribute.signup') }}{% endblock %}
```

Comment:
Do we still need this block? Isn't this the same URL specified in `taskview-base.html`?